### PR TITLE
PT-880 Display device parameters in Daily view

### PR DESCRIPTION
**Pull request by franck-fourel**
Groups changes of parameters by 30 minutes and display the group of changes in the daily view

### CODE REVIEW

**Comment by franck-fourel** [comment]
Below is an example of what it does
image

**Comment by franck-fourel** [comment]
Linked to 
- https://github.com/mdblp/viz/pull/45
- https://github.com/mdblp/tideline/pull/26

**Comment by yvon-dblg** [comment]
Some long parameters without space (not translated?) can make the view too small:
image

File: app/components/chart/daily.js [hunk 1]
```diff
@@ -657,6 +677,31 @@ class Daily extends Component {
...
   };
 
+  handleParameterHover = parameter => {
+    console.log('handleParameterHover');
+    console.log(parameter);
+    var rect = parameter.rect;
+    // range here is -12 to 12
+    var hoursOffset = sundial.dateDifference(parameter.data.normalTime, this.state.datetimeLocation, 'h');
+    parameter.top = rect.top + (rect.height / 2)
+    if(hoursOffset > 5) {
```
Comment (yvon-dblg) [inline app/components/chart/daily.js]:
```suggestion
    if (hoursOffset > 5) {
```

Comment (yvon-dblg) [inline app/components/chart/daily.js]:
```suggestion
    if (hoursOffset > 5) {
```

**Comment by franck-fourel** [comment]
> Some long parameters without space (not translated?) can make the view too small:
> image

yep but it's not supposed to happen ;-) 
we can make the tooltip larger if needed

**Pull request merged**